code/_core/obj/item/fishing/rod.dm

```diff
@@ -17,6 +17,7 @@
 	var/obj/item/fishing/bait/bait
 
 	var/obj/effect/fishing_bob/fishing_bob
+	var/obj/effect/alert/exclaim/fishing_alert
```

Comment:
It would be better if you just made this a temp effect so you don't have to store the alert effect here.

Reply:
not sure it'd be good as its currently called by multiple functions so it can get deleted as soon as it is reeled in and such as they stay for a couple seconds, does this require a rewrite or no?

Reply:
I guess your code is fine.